### ISSUE TITLE
Window repositions itself upon resize

### ISSUE THREAD

**Issue by ghost** [mikaa123/linear#30]
Linear frequently and seemingly arbitrarily finds itself in a state in which its window is repositions itself upon being resized. Restarting does not resolve the issue; it simply goes away after a while. 

Here is a gif: [http://g.recordit.co/AWZkd78L8I.gif](http://g.recordit.co/AWZkd78L8I.gif).

**Comment by radiovisual** [comment]
Strange. This has never happened to me. Is there anything else you can tell us that might help us figure out what is going on?

**Comment by ghost** [comment]
No, I can't think of anything useful to add. It happens quite sporadically. I looked for an open Electron issue for it but didn't find one.

**Comment by mikaa123** [comment]
Strange indeed! It seems like the x coordinate snaps back to some offset. Linear delegates all the window manipulation (e.g. resize) to Electron. It's basically an Electron window floating around.

I'll try bumping the electron version to see if it fixes it.

**Comment by mikaa123** [comment]
There was actually a pretty big version gap: went from `0.35.4` to `1.2.2`. I hope it fixes the issue. I made a new release: https://github.com/mikaa123/linear/releases/tag/1.4.0.

**Comment by ghost** [comment]
I'll give it a try. The issue sometimes takes a few days to happen.

**Comment by ghost** [comment]
With 1.4.0, Linear no longer launches, even after a restart.

**Comment by radiovisual** [comment]
there is likely some out-dated code, or calls to deprecated APIs with such a major version gap.

**Comment by mikaa123** [comment]
Weird, it works on my machine. I'll give it a closer look.

**Comment by mikaa123** [comment]
How about now: https://github.com/mikaa123/linear/releases/tag/1.4.1 🔥

**Comment by ghost** [comment]
No, still crashes.

**Comment by radiovisual** [comment]
@clintonbloodworth , what version of Mac OS X are you using?

**Comment by radiovisual** [comment]
Also, @clintonbloodworth, can you confirm which installation method you used to get the new version installed on your machine?
- [ ] installing via homebrew
- [ ] installing via zip download/extract
- [ ] installing via git clone/npm install?

**Comment by ghost** [comment]
I'm using OS X 10.11.5.

I tried installing using Homebrew, and by extracting the zip and moving its contents to /Applications.

**Comment by radiovisual** [comment]
@clintonbloodworth , can you try to see if it runs or crashes if you clone/npm install:

```
$ git clone https://github.com/mikaa123/linear.git
$ cd linear && npm install && npm start
```

If it crashes, please attach any errors you see in the console.

**Comment by ghost** [comment]
It worked. And now Linear installed using Homebrew starts.

**Comment by mikaa123** [comment]
Awesome 👍 . Let us know if the resize problem comes back.

**Comment by ghost** [comment]
Thanks. I'll keep an eye on it.

**Comment by ghost** [comment]
Unfortunately, it's still happening. 

In fact, I tried reinstalling 1.4.1 (just to be sure I was using 1.4.1) and, after doing so, Linear no longer launches: http://recordit.co/7Zf7F62PEH. Reverting to 1.3.0 solves this.

**Comment by radiovisual** [comment]
@clintonbloodworth , we still need to investigate why linear is not launching via the normal installation routines. After you removed `v1.4.1` you probably needed to run the same code suggested above:

```
$ git clone https://github.com/mikaa123/linear.git
$ cd linear && npm install && npm start
```

and then, for some mysterious reason, `v1.4.1` would work for you (even the version installed in the "normal" way).

**Comment by ghost** [comment]
Sorry for the delay. When I run:

`$ rm -rf ~/.linear`
`$ open /Applications/linear.app`

Linear crashes.

But when I run: 

`$ rm -rf ~/.linear`
`$ git clone https://github.com/mikaa123/linear.git`
`$ cd linear && npm install && npm start`
`$ kill $(ps aux | grep 'linear' | awk '{print $2}')`
`$ open /Applications/linear.app`

it doesn't. Though I've discovered that it doesn't work properly:
- It doesn't appear in the system menu bar.
- `tutorialShown` is not set to `true`, so the tutorial shows at every launch.
- The system dialog, `You are opening the application “linear” for the first time. Are you sure [...]`, appears at every launch. 

I tried running Linear from a new user account, but the results were the same.
